images/baremetal/Dockerfile.ci

```diff
@@ -11,11 +11,23 @@ RUN TAGS="libvirt baremetal" hack/build.sh
 
 FROM registry.svc.ci.openshift.org/origin/4.1:base
 COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
+COPY --from=builder /go/src/github.com/openshift/installer/images/libvirt/mock-nss.sh /bin/mock-nss.sh
 
 RUN yum update -y && \
-    yum install --setopt=tsflags=nodocs -y libvirt-libs && \
+    yum install --setopt=tsflags=nodocs -y \
+    libvirt-libs nss_wrapper openssl unzip jq openssh-clients && \
     yum clean all && rm -rf /var/cache/yum/*
 
+ENV TERRAFORM_VERSION=0.11.11
+RUN curl -O https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
+  unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /bin/ && \
+  rm terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+ENV MATCHBOX_VERSION=v0.2.3
+RUN curl -L -O https://github.com/poseidon/terraform-provider-matchbox/releases/download/${MATCHBOX_VERSION}/terraform-provider-matchbox-${MATCHBOX_VERSION}-linux-amd64.tar.gz && \
+  tar xzf terraform-provider-matchbox-${MATCHBOX_VERSION}-linux-amd64.tar.gz && \
+  mv terraform-provider-matchbox-${MATCHBOX_VERSION}-linux-amd64/terraform-provider-matchbox /bin/terraform-provider-matchbox && \
+  rm -rf terraform-provider-matchbox-${MATCHBOX_VERSION}-linux-amd64.tar.gz terraform-provider-matchbox-${MATCHBOX_VERSION}-linux-amd64
```

Comment:
I think we have to leave this terraform stuff in upi-installer and use that for packet setup/teardown. You should be able to add ssh over there.

I'd leave the other baremetal image changes and use it instead of libvirt-installer for devscripts-setup.

Reply:
Done